app/views/root/place.html.erb

```diff
@@ -10,17 +10,21 @@
 
         <%= raw @publication.introduction %>
 
-        <%= render :partial => 'location_form', :locals => {:format => 'service'} %>
+        <%= render partial: 'location_form', locals: { format: 'service' } %>
       </div>
     </div>
   </section>
 
   <% if @postcode.present? and !@publication.places.nil? %>
     <% if @publication.places.any? %>
-      <section class="places">
-        <h2>Results near <strong><%= @postcode %></strong>:</h2>
+      <section class="places"
+               data-module="auto-track-event"
+               data-track-category="placeSearch"
+               data-track-action="postcodeResultShown"
+               data-track-label="<%= places_names(@publication.places) %>">
```

Comment:
After talking to Tim (our analyst) we settled on setting the label to be the name of the first result. This addresses most of the concerns around this issue. See commit notes for [rebased commit](https://github.com/alphagov/frontend/commit/380eb25c60c5a2dae368df3128aa9a47cc0d97b3): 

"... 

We also add the name of the first places result as a label for GA tracking. This allows for specific tracking of shown results in the DfE campaign to promote the "Report child abuse" page. For non-report-child-abuse cases this is not necessarily useful, but it is deemed as the best compromise to allow tracking for this specific campaign and may be extended in future (by sending all results) for other formats if useful."